Documentation/ceph-common-issues.md

```diff
@@ -814,6 +814,8 @@ If you still decide to configure an OSD on LVM, please keep the following in min
 
 - Disable `lvmetad.`
 - Avoid configuration of LVs from the host. In addition, don't touch the VGs and physical volumes that back these LVs. 
-- Avoid incrementing the `count` field of `storageClassDeviceSets` and create a new LV that backs a OSD simultaneously.
+- Avoid incrementing the `count` field of `storageClassDeviceSets` and create a new LV that backs an OSD simultaneously.
 
 You can know whether the above-mentioned tag exists tag with the command: `sudo lvs -o lv_name,lv_tags`. If the `lv_tag` field is empty in an LV corresponding to the OSD lv_tags, this OSD encountered the problem. In this case, please [retire this OSD](ceph-osd-mgmt.md#remove-an-osd) or replace with other new OSD before restarting.
+
+Rook no longer uses `ceph-volume` lvm mode in a prepare job and creates OSDs with `ceph-volume` raw mode instead after merging the [PR](https://github.com/rook/rook/pull/6184). The existing lvm mode OSDs work continuously even if upgrade your Rook. However, using the raw mode OSDs is recommended because of the above-mentioned problem. You can replace the existing OSDs with raw mode OSDs by retiring them and adding new OSDs one by one. See the documents [Remove an OSD](ceph-osd-mgmt.md#remove-an-osd) and [Add an OSD on a PVC](ceph-osd-mgmt.md#add-an-osd-on-a-pvc).
```

Comment:
> Just to be very clear, Rook will always create all new OSDs in raw mode now, correct?

@travisn Yes in OSD on PVC.

> Were we waiting for a fix in ceph-volume to support this? I thought we were blocked on doing this. But the integration tests are passing, so it seems to be working on the latest nautilus and octopus.

Currently, `ceph-volume raw prepare` can create LVM mode OSD. However, `ceph-volume inventory` regards this LVM as unavailable. I'm working on fixing this problem. As a first step, the following PR will support crypt.

https://github.com/ceph/ceph/pull/34966

After merging this PR, I'll try to support LVM in raw mode OSD. Before that, Rook will call `ceph-volume raw prepare` without calling `ceph-volume inventory` as current cyrpt and mpath.

Reply:
Ok, I was confusing removing the "ceph-volume lvm" mode with detecting whether the PV is backed by LV. This PR is only changing OSD creation when on PVs that are backed by an LV so that we always use raw mode in that case, right? So this isn't as big a switch for all scenarios as I was thinking.

Reply:
> This PR is only changing OSD creation when on PVs that are backed by an LV so that we always use raw mode in that case, right? So this isn't as big a switch for all scenarios as I was thinking.

Correct.

Reply:
This statement still doesn't seem correct:
"Rook no longer uses `ceph-volume` lvm mode in a prepare job and creates OSDs with `ceph-volume` raw mode instead after merging the [PR](https://github.com/rook/rook/pull/6184)."

There are still some scenarios where lvm mode is used, such as for encrypted clusters.  I'd also suggest not mentioning the PR number, it doesn't matter after the merge.

Reply:
Thanks, fixed.